lib/gnat.ex

```diff
@@ -37,11 +37,16 @@ defmodule Gnat do
     end
   end
 
+  defp process_message({:msg, topic, sid, body}, state) do
+    send state.receivers[sid], {:msg, topic, body}
+  end
+  defp process_message("PING", state) do
```

Comment:
Why `"PING"` and not `:ping` or even `{:ping}`. Am I worried to much about symmetry?